src/node_sqlite.cc

```diff
@@ -67,6 +67,63 @@ using v8::Value;
     }                                                                          \
   } while (0)
 
+#define SQLITE_VALUE_TO_JS(                                                    \
+    from, unreachable_message, isolate, use_big_int_args, ...)                 \
```

Comment:
I would drop the `unreachable_message` parameter. It complicates this a bit and should never be encountered anyway.

Reply:
Done